docs/gpg.md

```diff
@@ -0,0 +1,29 @@
+# GPG Keys
+
+## Summary
+
+After our security incident in July 2014, we planned to try and contain the scope of our GPG keys to avoid resigning lots of content if (or rather, when) a key is compromised or has to be revoked.
+
+* General use time based keys: for use with Debian archives, nightly packages etc.  Cycled every two years.
+* Major release (1.5, 1.6 etc) based keys: for use with tarballs, RPMs.  Expiry of one year.
```

Comment:
I think these examples are technically minor releases. Perhaps we should refer to this as just `Release based keys` instead of `Major release ...`?

Reply:
Yeah, sounds good.